README.md

```diff
@@ -1 +1,32 @@
-# angular-hello-world-lab
\ No newline at end of file
+# Hello World 
+
+## Objectives
+
+- Create a Controller
+- Create an element for Controller binding
+- Create a public $scope method
+- Display the $scope value in the View
+
+## Instructions
+
+Setup the directory structure as follows:
+
+- js/
+  - app/
+    - controllers/
+  - angular.js
+- index.html
+
+You can find `index.html` and `angular.js` in this repo.
+
+Create a new module in `js/app/app.js`, named `app`.
+
+Create a controller named `MainController` inside `js/app/controllers/MainController.js` and attach it to a module named `app`. The function should take one parameter - `$scope`.
+
+In our HTML, initiate our module `app` using `ng-app` on a HTML element.
```

Comment:
I would combine these into one line and maybe preface the statement with "Don't forget to initiate our app and controller in the `index.html` file.  A small change, but it will make it read more like a helpful hint rather than a step-by-step list of instructions.